apps/spectator/app/game/[id]/_components/CheerTalk/_components/MenuModal/MenuModal.css.ts

```diff
@@ -1,17 +1,20 @@
 import { rem, theme } from '@hcc/styles';
 import { style } from '@vanilla-extract/css';
+import { calc } from '@vanilla-extract/css-utils';
 
 export const container = style({
   display: 'flex',
   flexDirection: 'column',
   width: '100%',
-  maxWidth: rem(218),
-  marginInline: 'auto',
-  paddingInline: theme.spaces.default,
+  maxWidth: calc.divide(theme.sizes.appWidth, 2),
+  margin: '0 auto',
+  padding: `0 ${rem(16)}`,
 });
 
 export const content = style({
-  padding: `${theme.spaces.xs} ${theme.spaces.default}`,
+  width: '100%',
+
+  padding: `${rem(8)} ${rem(16)}`,
```

Comment:
```suggestion
  paddingBlock: theme.spaces.xs
  paddingInline: theme.spaces.default
```

미리 지정해둔 테마를 이용하면 좋을 것 같습니다. block / inline으로 나눠 사용하는 것은 저였다면, 템플릿 리터럴을 피하기 위해 그렇게 했을 것 같지만 굳이 두 줄로 나눠 써야 하는 것이 불필요하다고 느끼신다면 값만 변경하셔도 좋습니다ㅎㅎ